### PR TITLE
Symintify TensorFactories.empty_like

### PR DESCRIPTION
**Pull request by vanbasten23**
Fixes [#ISSUE_NUMBER](https://github.com/pytorch/xla/pull/4876)

### CODE REVIEW

**Comment by vanbasten23** [comment]
Created a PR according to the comments in the [Pytorch/XLA PR](https://github.com/pytorch/xla/pull/4876).
@ezyang @bdhirsh can you please take a look when you get a chance?

cc @miladm @JackCaoG

File: aten/src/ATen/native/TensorFactories.cpp [hunk 1]
```diff
@@ -351,11 +351,11 @@ Tensor empty_like(
       result = at::empty_strided(self.sizes(), strides, options.memory_format(c10::nullopt));
     } else {
       // See Note [Explicit nullopt MemoryFormat argument]
-      result = at::empty(self.sizes(), options.memory_format(self.suggest_memory_format()), c10::nullopt);
+      result = at::empty_symint(self.sym_sizes(), options.memory_format(self.suggest_memory_format()), c10::nullopt);
     }
   } else {
     // See Note [Explicit nullopt MemoryFormat argument]
-    result = at::empty(self.sizes(), options.memory_format(memory_format), c10::nullopt);
+    result = at::empty_symint(self.sym_sizes(), options.memory_format(self.suggest_memory_format()), c10::nullopt);
```
Comment (ezyang) [inline aten/src/ATen/native/TensorFactories.cpp:358]: this change looks wrong

Comment (vanbasten23) [reply aten/src/ATen/native/TensorFactories.cpp]: fixed it.

**Review by ezyang — APPROVED** [review]

**Comment by vanbasten23** [comment]
Thanks Ed for the review!

**Pull request closed without merging**